src/test/java/pro/sky/java/course2/calculator/CalculatorServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package pro.sky.java.course2.calculator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+public class CalculatorServiceTest {
+
+
+    @Test
+    public void shouldReturnCorrectResultAfterPlus() {
+
+       int num1 = 3;
+       int num2 = 6;
+
+       int result = num1 + num2;
+       Assertions.assertEquals(9,result);
```

Comment:
лучше не использовать здесь переменные, обычное выражение сложения с числами проще будет прочитать. Аналогично и для других методов
```suggestion
       int result = 3 + 6;
       Assertions.assertEquals(9,result);
```